### PR TITLE
fix: cleanup template dockerfile

### PR DESCRIPTION
**Pull request by mdwcrft**
Small clean up of the template dockerfile to remove some unneeded steps

### CODE REVIEW

File: template.Dockerfile [hunk 1]
```diff
@@ -13,35 +13,18 @@ ENV PYTHONFAULTHANDLER=1 \
...
 
 # Copy the project to the system
 COPY iso15118/ iso15118/
 
-# Run the tests and linting
-COPY tests/ tests/
-RUN poetry run black --check --diff --line-length=88 iso15118 tests
-RUN poetry run flake8 --config .flake8 iso15118 tests
-# RUN poetry run mypy --config-file mypy.ini iso15118 tests
-RUN poetry run pytest -vv --cov-config .coveragerc --cov-report term-missing  --durations=3 --cov=.
```
Comment (tropxy) [inline template.Dockerfile:42]: why did you remove this?

**Review by tropxy — CHANGES_REQUESTED** [review]

File: template.Dockerfile (hunk 1 above)
Comment (mdwcrft) [reply template.Dockerfile:42]: I'm not sure it's good practice to run tests as part of a docker image build, the flow mentioned here is more standard: https://stackoverflow.com/questions/67088512/should-i-run-tests-during-the-docker-build

Comment (tropxy) [reply template.Dockerfile:42]: ok, I see the point, but if we remove it from the dockerfile then we need to provide an alternative to run the tests, otherwise there wont be running tests

Comment (mdwcrft) [reply template.Dockerfile:42]: Done in #147

**Comment by tropxy** [comment]
can you rebase?
